src/constants.ts

```diff
@@ -38,3 +38,29 @@ export const TRUSTED_PROXIES = [
       .join('')
   ),
 ];
+export const API = {
```

Comment:
I think there is no need to keep API endpoints names in the const.

Reply:
Do you think we should expose API name constants / enums anywhere or just get rid of them?